Sources/MeiliSearchCore/PackageVersion.swift

```diff
@@ -1,18 +1,15 @@
 import Foundation
-#if canImport(FoundationNetworking)
-  import FoundationNetworking
-#endif
 
-internal struct PackageVersion {
+public struct PackageVersion {
   /// This is the current version of the meilisearch-swift package
-  private static let current = "0.15.0"
+  public static let current = "0.15.0"
```

Comment:
question: why has the access level of this variable changed? where else is it used?